dist/pythonlibs/testrunner/__init__.py

```diff
@@ -44,3 +45,16 @@ def run(testfunc, timeout=TIMEOUT, echo=True, traceback=False):
         print("")
         teardown_child(child)
     return 0
+
+
+def check_unittests_func(child, timeout=TIMEOUT, nb_tests=None):
```

Comment:
The test is somewhat clunky... how about

```suggestion
def check_unittests_ok(child, timeout=TIMEOUT, nb_tests=None):
```

Reply:
This is marked as resolved before I addressed it. I don't like either the `check_unittests_ok` name so I went for another approach: use name `check_unittests` name in this case and use a `run_check_unittests` in the `main` block. This seems better to me.

Reply:
Marking it as resolved seem to have happened accidentally.